Initial inclusion of an ID card for GOCDB

gwarf opened the pull request:
Initial inclusion of an ID card for GOCDB.
It's from https://wiki.egi.eu/wiki/GOCDB

Content, naming and presentation to be reviewed/agreed.

![Screenshot 2021-02-18 at 18 30 57](https://user-images.githubusercontent.com/451417/108396736-78b6eb00-7217-11eb-8174-66cf8d6036ac.png)

gwarf commented:
@paolini78 I had a first test at adding an identity card, we need to discuss what is meaningful to be presented there.

enolfc commented:
Should we have one of these for every service?

gwarf commented:
> Should we have one of these for every service?

Yes it's the intent. It's to replace things we have in the wiki for the central services, like https://wiki.egi.eu/wiki/GOCDB

enolfc commented on content/en/internal/configuration-database/_index.md:
```diff
@@ -9,9 +9,36 @@ type: "docs"
+| Release notes                | [Release notes](https://github.com/GOCDB/gocdb/releases)              |
+| Issue tracker for developers | https://github.com/GOCDB/gocdb/issues                                 |
+| Source code                  | https://github.com/GOCDB                                              |
+| License                      | gLite/Apache 2                                                        |
```
github repo says Apache 2

gwarf replied:
Updated

enolfc commented on content/en/internal/configuration-database/_index.md:
```diff
@@ -9,9 +9,36 @@ type: "docs"
+| Issue tracker for developers | https://github.com/GOCDB/gocdb/issues                                 |
+| Source code                  | https://github.com/GOCDB                                              |
+| License                      | gLite/Apache 2                                                        |
+| Privacy Policy               | https://wiki.egi.eu/wiki/GOCDB/data_privacy                           |
```
What about this? https://goc.egi.eu/privacy.html

gwarf replied:
Updated

enolfc commented on content/en/internal/configuration-database/_index.md:
```diff
@@ -9,9 +9,36 @@ type: "docs"
+| Description                  | Central registry of the infrastructure topology                       |
+| URL                          | https://goc.egi.eu                                                    |
+| Support Email                | gocdb-admin@mailman.egi.eu                                            |
+| Helpdesk Support Unit        | EGI Services and Service components -> Configuration Database (GOCDB) |
```
Do we have a link for this?

gwarf replied:
If we had some basic stuff about using GGUS under https://docs.egi.eu/support/ we could point to that.
I don't think (may be wrong, never saw it but never tried) we can easily make an URL allowing to create a ticket for a specific SU.

gwarf replied:
I've added a link to the Helpdesk page (that is a bit rough)

enolfc reviewed (commented):
Some minor things

gwarf commented:
> Some minor things

Update screenshot addressing comments.
![Screenshot 2021-03-29 at 14 21 58](https://user-images.githubusercontent.com/451417/112835926-37c5b800-909a-11eb-8004-7b497e1534f5.png)

enolfc reviewed (approved):
LGTM!
